### PR TITLE
sbs: turn on SBS by default

### PR DESCRIPTION
**Pull request by strib**
No need for a config option anymore.

### CODE REVIEW

**Comment by strib** [comment]
(Just getting this ready, not going to push it until we test.)

**Comment by strib** [comment]
The basic tests work.  I probably won't push this until we get some clarity on the docker offline-device issue, but might as well get it reviewed in the meantime.

File: libkbfs/tlf_handle_test.go [hunk 1]
```diff
@@ -268,11 +268,7 @@ func TestParseTlfHandleSocialAssertion(t *testing.T) {
 	}
 
 	name := "u1,u2#u3@twitter"
-	_, err := ParseTlfHandle(ctx, kbpki, name, false, false)
-	assert.Equal(t, 0, kbpki.getIdentifyCalls())
-	assert.Equal(t, NoSuchUserError{"u3@twitter"}, err)
-
-	h, err := ParseTlfHandle(ctx, kbpki, name, false, true)
+	h, err := ParseTlfHandle(ctx, kbpki, name, false)
```
Comment (akalin-keybase) [inline libkbfs/tlf_handle_test.go]: h -> _

**Comment by akalin-keybase** [comment]
lgtm

File: libkbfs/tlf_handle_test.go (hunk 1 above)
Comment (strib) [inline libkbfs/tlf_handle_test.go]: Why?  it's used in `h.GetCanonicalName()` below.

Comment (akalin-keybase) [inline libkbfs/tlf_handle_test.go]: Nevermind, I misread the diff!

**Pull request merged**